packages/newsroom-signup/src/Newsroom.tsx

```diff
@@ -506,11 +533,22 @@ class NewsroomComponent extends React.Component<NewsroomProps & DispatchProp<any
   };
 }
 
-const mapStateToProps = (state: StateWithNewsroom, ownProps: NewsroomExternalProps): NewsroomReduxProps => {
+const mapStateToProps = (state: StateWithNewsroom, ownProps: NewsroomGqlProps): NewsroomReduxProps => {
   const { newsroomAddress } = ownProps;
   const newsroom = state.newsrooms.get(newsroomAddress || "") || { wrapper: { data: {} } };
+  const { user, parameters } = (state as any).networkDependent;
```

Comment:
I don't know what to do about this, but this seems very wrong.  Reaching up out of this package into the parent context this way.

Reply:
can this be passed in through root component

Reply:
This is a good point, though we already do this in various spots in the tutorial, token purchase, and apply to TCR steps (e.g. https://github.com/joincivil/Civil/blob/a3bce8d935c4d9bc2d107132c3f823075999942d/packages/newsroom-signup/src/ApplyToTCR/index.tsx#L93)

Passing them in through root component would work but would have to be passed down the tree to a dozen child components. What we probably should do is add these to the Civil context or a new context.

It's a small-to-medium refactor, I think we should punt on this until such time that newsroom-signup appears in any other context or at least next time we need to use these values - would like to get this PR out since there are a few support threads going that this should help with.

Reply:
i like the idea of using a context.  Can you leave a comment with a reminder about that or make a ticket

Reply:
Yep will do